src/backend/distributed/ddl/foreign_constraint.c

```diff
@@ -649,3 +649,63 @@ HeapTupleOfForeignConstraintIncludesColumn(HeapTuple heapTuple, Oid relationId,
 
 	return false;
 }
+
+
+/*
+ * This function will be removed later
+ */
+List *
+GetReferencedReferenceTableList(Oid relationId)
+{
+	List *tableForeignConstraints = NIL;
+
+	Relation pgConstraint = NULL;
+	SysScanDesc scanDescriptor = NULL;
+	ScanKeyData scanKey[1];
+	int scanKeyCount = 1;
+	HeapTuple heapTuple = NULL;
+
+	/*
+	 * Set search_path to NIL so that all objects outside of pg_catalog will be
+	 * schema-prefixed. pg_catalog will be added automatically when we call
+	 * PushOverrideSearchPath(), since we set addCatalog to true;
+	 */
+	OverrideSearchPath *overridePath = GetOverrideSearchPath(CurrentMemoryContext);
+	overridePath->schemas = NIL;
+	overridePath->addCatalog = true;
+	PushOverrideSearchPath(overridePath);
```

Comment:
why are we changing the search path here? there's no deparsing in this function afaik